src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/review/ReviewScheduleEntity.kt

```diff
@@ -30,24 +30,24 @@ data class ReviewScheduleEntity(
   val prisonNumber: String,
 
   @Column(updatable = false)
-  val earliestReviewDate: LocalDate,
+  var earliestReviewDate: LocalDate,
```

Comment:
These 5 fields need to be `var` in the entity class, so that we can update an existing entity with new values